libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -655,13 +655,15 @@ bool NavEKF2::InitialiseFilter(void)
             } else {
                 initFailure = InitFailures::NO_MASK;
             }
+            num_cores = 0;
```

Comment:
You don't need this one.

Reply:
true, but harmless

Reply:
I would argue that it makes the code harder to understand. The presumption is that the code is there for a reason, if that reason is not obvious it is confusing. EG: Why would num_core have changed and need resetting in the four lines since we checked it.

Reply:
changed